renovate.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "baseBranches":[
+     "main",
+     "release-v0.20",
+     "release-v0.19",
+     "release-v0.18",
+     "release-v0.17"
+  ],
+  "constraints":{
+     "go":"1.22"
```

Comment:
Older branches use older go versions. Can we configure this constraint for each branch?